SConstruct

```diff
@@ -334,6 +334,10 @@ config_options = [
         'prefix',
         'Set this to the directory where Cantera should be installed.',
         defaults.prefix, PathVariable.PathAccept),
+    PathVariable(
+        'libdirname',
+        'Set this to the directory where Cantera libraries should be installed.',
+        '', PathVariable.PathAccept),
```

Comment:
The default value of `lib` can be specified here, rather than needing the `if` block further down. I would also move the description of why this option exists and the examples of where its value differs from the default to this docstring.

Reply:
@speth, thank you for review. 
I tried to change the commit by appropriate way.